tools/target.py

```diff
@@ -1,5 +1,6 @@
 #!/usr/bin/env python3
 
+import os
```

Comment:
Shouldn't all the changes in this file be in another PR?

Reply:
You're right. They have been merged to master. I merged master into my branch off his branch, without merging it into his branch. Rookie error on my part lel